scripts/install.sh

```diff
@@ -5,68 +5,68 @@ OS="$(uname -s)"
 }
 
 unixlike_add_path() {
-  echo "Attempting to add path: $1"
-  rcpath=""
-  if [ "${SHELL}" = "/bin/bash" ]; then
-    rcpath="${HOME}/.bashrc"
-  elif [ "${SHELL}" = "/bin/zsh" ]; then
-    rcpath="${HOME}/.zshrc"
-  fi
+	echo "Attempting to add path: $1"
+	rcpath=""
+	if [ "${SHELL}" = "/bin/bash" ]; then
+		rcpath="${HOME}/.bashrc"
+	elif [ "${SHELL}" = "/bin/zsh" ]; then
+		rcpath="${HOME}/.zshrc"
+	fi
 
-  if [ "${rcpath}" = "" ]; then
-    echo "Cannot autodetect shell (only BASH and ZSH are currently supported)"
-    exit 1
-  fi
+	if [ "${rcpath}" = "" ]; then
+		echo "Cannot autodetect shell (only BASH and ZSH are currently supported)"
+		exit 1
+	fi
 
-  cat "${rcpath}" | grep -q "$1" || \
-    echo -e "export PATH=\"$1:\${PATH}\"\n" >> "${rcpath}"
+	cat "${rcpath}" | grep -q "$1" ||
+		echo -e "export PATH=\"$1:\${PATH}\"\n" >>"${rcpath}"
 }
 
 install_for_macos() {
-  echo "Checking for Homebrew..."
-  # See https://stackoverflow.com/a/26759734/1156132
-  if ! [ -x "$(command -v brew)" ]; then
-    echo "Installing Homebrew..."
-    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-  fi
+	echo "Checking for Homebrew..."
+	# See https://stackoverflow.com/a/26759734/1156132
+	if ! [ -x "$(command -v brew)" ]; then
+		echo "Installing Homebrew..."
+		/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+	fi
 
-  echo "Installing requirements through Homebrew..."
-  brew install git node@12 pandoc tectonic graphicsmagick ghostscript || true
-  # Ensure our path is set up correctly to use the freshly installed NodeJS
-  unixlike_add_path "/usr/local/opt/node@12/bin"
-  export PATH="/usr/local/opt/node@12/bin:${PATH}"
+	echo "Installing requirements through Homebrew..."
+	brew install git node@12 pandoc tectonic graphicsmagick ghostscript pandoc-crossref || true
+	# Ensure our path is set up correctly to use the freshly installed NodeJS
+	unixlike_add_path "/usr/local/opt/node@12/bin"
+	export PATH="/usr/local/opt/node@12/bin:${PATH}"
 
-  # Update NPM and install Yarn
-  npm i -g npm yarn
+	# Update NPM and install Yarn
+	npm i -g npm yarn
 
-  echo "Cloning repository..."
-  rm -rf /tmp/themis-contract
-  git clone ${THEMIS_CONTRACT_REPO} /tmp/themis-contract
+	echo "Cloning repository..."
+	rm -rf /tmp/themis-contract
+	git clone ${THEMIS_CONTRACT_REPO} /tmp/themis-contract
 
-  echo "Uninstalling any old versions of themis-contract..."
-  npm uninstall -g themis-contract || true
-  yarn global remove themis-contract || true
+	echo "Uninstalling any old versions of themis-contract..."
+	npm uninstall -g themis-contract || true
+	yarn global remove themis-contract || true
 
-  echo "Installing themis-contract..."
-  cd /tmp/themis-contract && \
-  yarn install && \
-  yarn global add file:`pwd`
+	echo "Installing themis-contract..."
+	cd /tmp/themis-contract &&
+		yarn install &&
+		yarn global add file:$(pwd)
```

Comment:
Should this not maybe be:

```suggestion
		yarn global add file://$(pwd)
```

Does it matter? I usually install it globally referencing it as a fully qualified URL.

Reply:
You're welcome to revert them, or keep them, overall it doesn't seem to change things substantially. `$(pwd)` may be more idiomatic, I'm not totally sure, and it's equivalent to the first version anyways.

My concern's more around how `yarn global add` works, and how the first version may not have been accurate by leaving out the `//`.

Reply:
Ah ha. got it. I know literally nothing about how yarn works :) Including why we use yarn instead of npm in this script :D 

I'd say, if the current installation script is working as expected, maybe leave it as is? We ca always followup with a change for this if we find there are problems.

re: `$(...)`, my understanding is is preferred because of improved portability (used in all POSIX compliant shells) and cleaner semantics (see https://unix.stackexchange.com/a/5782/95793) But I wouldn't fuss with this stuff in my own reviews :)

I'm gonna revert. If we want to put the script under auto formatting, it can just be its own pr :) Sorry for the noise resulting.

Reply:
Sure, sounds good! 👍